tests/integration/it_property_resource_test.py

```diff
@@ -0,0 +1,42 @@
+""" Tests for PropertyResource, resource for internationalization based on a
+property file
+
+"""
+import pytest
+
+from timeless.message.property_resource import PropertyResource
+
+
+class TestPropertyResource():
+
+    directory = "resources/messages/"
+
+    @pytest.mark.skip(reason="PropertyResource not implemented yet")
+    def test_get_found(self):
+        """ Test if PropertyResource can return a value that exists"""
+        assert (
+            PropertyResource(
+                 directory=self.directory
+            ).get(self, "foundkey", "en_US") == "thevalue"
```

Comment:
@paulodamaso I think ``locale`` should be passed in the ctor and a single file be read/loaded per object. If you make it like this, this class might as well be a global Singleton since it will always load the required file at this method call.

Reply:
@amihaiemil I agree with you on this one, thanks